src/components/inline_edit/inline_edit_form.styles.ts

```diff
@@ -21,5 +21,12 @@ export const euiInlineEditReadModeStyles = ({ euiTheme }: UseEuiTheme) => {
         user-select: text;
       }
     `,
+
+    hasPlaceholder: css`
+      .euiText,
+      .euiTitle {
+        color: ${tint(euiTheme.colors.subduedText, 0.15)};
```

Comment:
axe is complaining that this is too low of color contrast (3.9 ratio, needs to be 4.5) on light mode - we'll likely need to change the tinting ratio based on color mode.

```suggestion
        color: ${colorMode === 'DARK' ?
          ? tint(euiTheme.colors.subduedText, 0.1)
          : tint(euiTheme.colors.subduedText, 0.2)
        };
```

(I think the dark mode could probably be a little lighter, hence my extra tweak there).

Or alternatively... I wonder if `transparentize` might be a better way to go that adjusts accordingly to more background types:

```suggestion
        color: ${transparentize(euiTheme.colors.subduedText, 0.15)};
```

Reply:
For `tint`,  `color: ${tint(euiTheme.colors.subduedText, 0.08)};` passes all tests in both light mode and dark mode. 
I don't think `transparentize` would help us in dark mode. Even with the alpha channel set to `1`, Axe throws errors for the color of the inline edit - text placeholder.

Reply:
Right, yeah, I was thinking more for possible scenarios where `EuiInlineEdit` is used against backgrounds that aren't just the default body background - axe still might throw errors in that case. No worries either way, we can help people figure that out if they run into it in the future 👍

Reply:
>

> For `tint`, `color: ${tint(euiTheme.colors.subduedText, 0.08)};` passes all tests in both light mode and dark mode.

Sorry to be nitpicky on this one - I think the color isn't subtle/different enough on dark mode to mark it clear that it's placeholder text, hence why I wanted to try increasing the tint color there. 🤔 

Just to double check also, is `.1` still failing axe in light mode? I assume so if you had to go all the way down to `.08`, just wanted to double check 👍

Reply:
Actually, after re-reviewing staging, I think the `.08` is just fine in both light and dark mode! Maybe my eyes were just going when I looked at it before. Although as a disclaimer, I don't have the discerning eyes of a designer 😅 

I re-checked axe in both light and dark mode again, and no color contrast warning this time 🎉